webapp/src/templates/partials/header.html

```diff
@@ -92,17 +92,16 @@
           </li>
 
           <li role="separator" class="divider" ng-if="!replicationStatus.disabled"></li>
-          <li role="presentation disabled" ng-show="!replicationStatus.disabled && replicationStatus.current !== 'unknown'">
-            <a role="menuitem" tabindex="-1" ng-if="replicationStatus.current !== 'in_progress'" ng-click="replicate()">
+          <li role="presentation disabled" ng-if="!replicationStatus.disabled && !replicationStatus.current.debounce">
```

Comment:
Is `debounce` just controlling whether or not to show the button which triggers replication? Can we think of another name for this, I don't think debounce is accurate here.

Reply:
My understanding of a debounce is a function which can only be called per time interval. Similar end result, but in my mind a debounce is a specific implementation. Could be wrong or outdated.